src/main/java/com/blackducksoftware/integration/jira/workflow/JiraVersionCheck.java

```diff
@@ -39,7 +39,7 @@
  * @author sbillings
  */
 public class JiraVersionCheck {
```

Comment:
This class should probably be deleted since we handle the exact same functionality in the marketplace.